Resources/templates/CommonAdmin/ActionsAction/ActionsBuilderAction.php.twig

```diff
@@ -24,6 +24,11 @@ class ActionsController extends BaseController
      */
     public function objectAction(Request $request, $pk, $action)
     {
+        // this is needed for confirmed object actions
+        if ($action == 'edit' || $action == 'show') {
+            return $this->redirectToRoute('{{ builder.baseActionsRoute }}_'.$action, array('pk' => $pk));
+        }
+
```

Comment:
i think yes, for example: on show page you have edit button which needs confirmation, object actions do POST...